doc/CollisionSolverEntities.doc.md

```diff
@@ -0,0 +1,43 @@
+# Collision Solver for Entities other than Isaac
+
+## Label
+
+`collision`

```

Comment:
The label is the label used to call the function, here I think it's `solve_collision_to_Isaac`?
Anyway `collusion` is used by the collision function.